src/Symfony/Component/VarDumper/Tests/Caster/DateCasterTest.php

```diff
@@ -173,6 +173,8 @@ public function provideIntervals()
             array('P5M', 0, '+ 5m', null),
             array('P6Y', 0, '+ 6y', null),
             array('P1Y2M3DT4H5M6S', 0, '+ 1y 2m 3d 04:05:06'.$ms, null),
+            array('PT3600S', 0, '+ 01:00:00'.$ms, null),
+            array('P60D', 0, '+ 60d', null),
```

Comment:
can you add an example with 1 month + 32 days? I would expect an output that says "1 month + 32 days" :)